### PR TITLE
Box or Many apps together

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Comment by Abdur-rahmaanJ** [comment]
@rehmanis idk why travis tests failing

File: shopyo/conftest.py [hunk 1]
```diff
@@ -94,15 +102,19 @@ def db_session(db):
...
     transaction.rollback()
     connection.close()
     session.remove()
 
 
+@pytest.fixture(scope="session")
+def _db(init_database):
+    return init_database
+
+
```
Comment (rehmanis) [inline shopyo/conftest.py]: Looks like the merge messed up some things. This function should not be there according to my recent PR. Check and compare the conftest.py file here https://github.com/rehmanis/shopyo/blob/3afe2e14c978854239130cfeadea547b89582ff8/shopyo/conftest.py

File: shopyo/modules/admin/test_user.py [hunk 2]
```diff
@@ -13,20 +15,21 @@ def test_new_user(new_user):
     assert not new_user.is_admin
 
 
-@pytest.mark.order('first')
-def test_home_page(test_client, db):
+@pytest.mark.order("first")
+def test_home_page(test_client, init_database):
+
```
Comment (rehmanis) [inline shopyo/modules/admin/test_user.py]: The removed changes(red ones) are the correct ones. Check the line 17 here https://github.com/rehmanis/shopyo/blob/3afe2e14c978854239130cfeadea547b89582ff8/shopyo/modules/admin/test_user.py

File: shopyo/modules/category/test_category.py [hunk 3]
```diff
@@ -16,27 +19,27 @@ def test_category_dashboard_page(test_client):
...
     assert b"Successfully logged out" in response.data
 
     # check request to category correctly redirects to login page
     response = test_client.get(
-        url_for('category.dashboard'), follow_redirects=True)
-    assert response.status_code == 200
-    assert request.path == url_for('login.login')
+        url_for("category.dashboard"), follow_redirects=True
+    )
+    assert request.path == url_for("login.login")
```
Comment (rehmanis) [inline shopyo/modules/category/test_category.py]: The remove changes are the correct one. It missing one assert line 26  `assert response.status_code == 200`

File: shopyo/conftest.py [hunk 4]
```diff
@@ -94,15 +102,19 @@ def db_session(db):
     options = dict(bind=connection, binds={})
     session = db.create_scoped_session(options=options)
 
-    db.session = session
-
+    yield db  # this is where the testing happens!
     yield session
```
Comment (rehmanis) [inline shopyo/conftest.py]: This is also messed up.  Correct one is here https://github.com/rehmanis/shopyo/blob/3afe2e14c978854239130cfeadea547b89582ff8/shopyo/conftest.py

**Review by rehmanis — CHANGES_REQUESTED** [review]

**Comment by rehmanis** [comment]
> @rehmanis idk why travis tests failing

conftest.py file is modified incorrectly after merge. Test cases are using old fixture `init_database` instead of `db`. I have pointed out the difference but might have missed some.

**Comment by Abdur-rahmaanJ** [comment]
I wrongly updated after merging your PR

**Comment by Abdur-rahmaanJ** [comment]
@rehmanis i just remerged your last branch (cherry-pick) into box, fixing the wrong ends on my part.

**Comment by rehmanis** [comment]
> @rehmanis i just remerged your last branch (cherry-pick) into box, fixing the wrong ends on my part

@Abdur-rahmaanJ looks like it is fine now. Now just missing 3 asserts as can be seen here https://github.com/Abdur-rahmaanJ/shopyo/pull/236/commits/0fbdfb4fa205bb6441139dd9757e5f4c1110f65a

However, I see a mistake in my code where I forgot to add the assert in line 57 and 182 in above referenced commit. If it is too much hassle just merge it and I will fix it in my next commit as these asserts are not critical.

**Comment by Abdur-rahmaanJ** [comment]
@rehomanis i did not cherry-pick the mentionned commit. Now supposedly ok. The mess was caused by me trying to find a way to force merge without having to resolve merge conflict

**Pull request merged**